templates/admin/auth/source/ldap.tmpl

```diff
@@ -79,21 +79,29 @@
 		<label for="attribute_avatar">{{.i18n.Tr "admin.auths.attribute_avatar"}}</label>
 		<input id="attribute_avatar" name="attribute_avatar" value="{{.attribute_avatar}}" placeholder="e.g. jpegPhoto">
 	</div>
+	<div class="inline field">
+		<div class="ui checkbox">
+			<label for="team_group_map_enabled"><strong>{{.i18n.Tr "admin.auths.team_group_map_enabled"}}</strong></label>
+			<input id="team_group_map_enabled" name="team_group_map_enabled" type="checkbox" class="ldap_group_options" {{if .team_group_map_enabled}}checked{{end}}>
```

Comment:
we had better keep CSS class names in keba-case, just like `security-protocol` and `use-paged-search` in this page.

Reply:
I updated the CSS class names.